config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Configuration for running the complete pipeline including modeling and evaluation.
+
+# Absolute paths to non-code resources.
+file_paths:
+  queries: "/home/jupyter/github_msba_capstone/msba-capstone-2020/queries"
+
+# Global config used by multiple pipeline steps. Also made available
+# as parameters to all SQL queries.
+global:
+  # Project and dataset configuration.
+  source_project_id: "bigquery-public-data"
+  destination_project_id: "ut-goog"
+  source_dataset: "cfpb_complaints"
+  destination_dataset: "cfpb_complaints"
+
+  # Clean data, minor refined versions of provided data.
+  source_table: "complaint_database"
+  nlp_features_table: "complaints_nlp"
+  cleaned_features_table: "complaints_clean_categories"
+  nulls_removed_table: "complaints_nulls_removed"
+  clean_table: "complaints_clean"
+  
+  # Split features into training and prediction.
+  train_predict_split: "TrainPredictSplit"
+  features_train_table: "FeaturesTrain"
+  features_predict_table: "FeaturesPredict"
+
+  # Output tables for predictions and failed predictions from the batch
+  # prediction job, ex. numeric column recieved a string.
+  predictions_table: "Predictions"
+  failed_predictions_table: "FailedPredictions"
+  
+  # Evaluation
+  eval_table: "Evaluation"
+  eval_metrics_table: "EvaluationMetrics"
+
+  # AutoML Parameters. It is assumed that no other datasets or models have
+  # been created with these names, or the pipeline will fail. They can be
+  # deleted using the client, or the UI.
+  dataset_display_name: "CFPBDataset"
+  model_display_name: "CFPBModel"
+  automl_compute_region: "us-central1"
+
+  # Final model will be trained on test data, metrics are currently based
```

Comment:
Does this comment maybe belong somewhere else? Maybe with the train table names? Or the splits? I think this may be an artifact of the original config file? It doesn't quite make sense here.

Reply:
Deleted the comment.